cmake/modules/ddr/cmake_ddr.awk

```diff
@@ -142,12 +142,13 @@ NR == 1 {
 {
 	# replace tabs with spaces, and cut multiple spaces
 	sub(/[ \t]+/, " ")
```

Comment:
This needs to use `gsub` to fix all places with two or more whitespace characters.